test/local/routes/sms.js

```diff
@@ -589,3 +589,103 @@ describe('/sms/status with disabled geo-ip lookup', () => {
   })
 })
 
+describe('/sms/status with query param and enabled geo-ip lookup', () => {
```

Comment:
In terms of paths through the code, the `REGIONS.has` has check is independently variable to the query param check, so I see it as already covered.

I know some people prefer a comprehensive/completionist approach though, were you asking from that perspective? I can add one if so.